src/test/java/rainy/tasks/TaskTest.java

```diff
@@ -0,0 +1,36 @@
+package rainy.tasks;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import rainy.rainyexceptions.InvalidMarkAndUnmarkException;
+
+
+public class TaskTest {
+
+    @Test
+    public void markTest() throws InvalidMarkAndUnmarkException {
```

Comment:
Perhaps you can consider using the following three part format `featureUnderTest_testScenario_expectedBehavior()` to name your test cases in a more descriptive manner?